src/ro/catalog/utilizatori/Profesor.java

```diff
@@ -0,0 +1,66 @@
+package ro.catalog.utilizatori;
+
+import ro.catalog.Materie;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Profesor extends Utilizator {
+    private List<Materie> materiiPredate = new ArrayList<>();
+
+    public Profesor(String nume, String prenume) {
+        super(nume, prenume);
+    }
+
+    public Profesor(String nume, String prenume, List<Materie> materiiPredate) {
+        super(nume, prenume);
+        this.materiiPredate = materiiPredate;
+    }
+
+    public String getNume() {
+        return nume;
+    }
+
+    public void setNume(String nume) {
+        this.nume = nume;
```

Comment:
since you inherit nume si prenume, why do you redeclare setters and getters for them?